subsys/net/lib/azure_fota/azure_fota.c

```diff
@@ -0,0 +1,565 @@
+/*
+ * Copyright 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
```

Comment:
Should contain a link to https://docs.microsoft.com/en-us/azure/iot-hub/tutorial-firmware-update because this implementation is based on the process outlined in that document.